sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.0.0-beta.2 (Unreleased)
 
+- Bug fix: Replaced cases where `instanceof` was used to compare errors with comparisons on the error names. Uneven updates on our dependencies would cause `instanceof` to mismatch our error comparisons. Comparing by error name makes sure dependency changes won't alter our logic.
```

Comment:
Can you add a specific example?

Like.. 

> Previously, if the msal@1.4.9 version is installed, "ClientAuthError" is thrown instead of "AuthError" in some cases whereas, for msal@1.4.8, you'd see the same "AuthError". And this PR #14763 attempts to make the errors returned consistent across versions.


Or something in those lines?

Reply:
Based on this: https://github.com/Azure/azure-sdk-for-js/pull/14763#discussion_r608242323 I'll remove the changelog entry.